iati/validator.py

```diff
@@ -290,32 +356,29 @@ def _check_codes(dataset, codelist):
         base_xpath = mapping['xpath']
         condition = mapping['condition']
         split_xpath = base_xpath.split('/')
-        parent_el_xpath = '/'.join(split_xpath[:-1])
-        attr_name = split_xpath[-1:][0][1:]
+        last_xpath_section = split_xpath[-1:][0]
```

Comment:
Same as https://github.com/IATI/pyIATI/pull/229/files#r149662369 goes for this line.

Reply:
d12d03fc196406326b43fd59f1c1fc366d558b4b